druid-filtering-ext/pom.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>bi.deep</groupId>
+  <artifactId>druid-filtering-ext</artifactId>
+  <name>Druid Filtering Extension</name>
+  <description>Filtering extension for Apache Druid.</description>
+
+  <parent>
+    <groupId>deep.bi</groupId>
+    <artifactId>deep-druid-extensions</artifactId>
+    <version>31.0.0-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+
+  <properties>
+    <license.file.path>../codestyle/LICENSE.txt</license.file.path>
+  </properties>
+
+  <scm>
+    <connection>scm:git:git://github.com/deep-bi/deep-druid-extensions.git</connection>
+    <developerConnection>scm:git:git://github.com/deep-bi/deep-druid-extensions.git</developerConnection>
+    <url>https://github.com/deep-bi/deep-druid-extensions/tree/main/druid-filtering-ext</url>
+    <tag>HEAD</tag>
+  </scm>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.druid</groupId>
+      <artifactId>druid-server</artifactId>
+      <version>${druid.version}</version>
+      <scope>provided</scope>
+    </dependency>
+
+    <dependency>
```

Comment:
Nit: This better to be consistent, we have junit4 in our parent pom already, i suggest moving everything to JUnit5

Reply:
Done